src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -1,54 +1,60 @@
+// All praise 254 lib
+
 package frc.robot.subsystems.vision;
 
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.subsystems.vision.VisionConstants.Limelight;
 import org.littletonrobotics.junction.AutoLogOutput;
 import org.littletonrobotics.junction.Logger;
 
 public class Vision extends SubsystemBase {
```

Comment:
Why did we move away from adding `Subsystem` after the names of our subsystems?

Reply:
bc visionsubsystem is icky